.changeset/four-waves-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': patch
+---
+
+The `AnyStateNode` type, which represents a `StateNode` of any context, event, etc., has been introduced.
```

Comment:
This is a somewhat internal change - I'm not sure how relevant this is for our users and I would probably skip "documenting" it:
```suggestion
---
---
```